### DISCUSSION TITLE
Solcon IPTV not working

### DISCUSSION THREAD

**Discussion by jaccoh** [fabianishere/udm-iptv#263] [General]
As of today I have Solcon internet and also IPTV ofcourse. I have managed to get internet up and running easily via my UDM Pro. 

Also I have followed your guide and used the install script. I am using this config:

```
## Example configuration for udmp-iptv ##
# Interface on which IPTV traffic enters the router
IPTV_WAN_INTERFACE="eth8"
# ID of VLAN which carries IPTV traffic (use 0 if no VLAN is used)
IPTV_WAN_VLAN="188"
# Name of the IPTV VLAN interface
IPTV_WAN_VLAN_INTERFACE="iptv"
# IP ranges from which the IPTV traffic originates (separated by spaces)
IPTV_WAN_RANGES="10.0.0.0/8  10.252.0.0/16  10.253.0.0/16  217.166.0.0/16"
# DHCP options to send when requesting an IP address
IPTV_WAN_DHCP_OPTIONS="-O staticroutes -V IPTV_RG"
# LAN interfaces on which IPTV should be made available
IPTV_LAN_INTERFACES="br188"
# Disable quickleave for igmpproxy
IPTV_IGMPPROXY_DISABLE_QUICKLEAVE="false"
# Enable debugging for igmpproxy
IPTV_IGMPPROXY_DEBUG="false"
IPTV_IGMPPROXY_PROGRAM="igmpproxy"

IPTV_IGMPPROXY_IGMP_VERSION="2"
```

But the Arris device from Solcon gives me an error "Error 4: Could not download the system software". With tcpdump I do see some bootp attempts. So I'm guessing thats what is failing.

Any pointers?

I did create a VLAN (188) on my LAN and that matches the VLAN Solcon provides (188).

**Comment by jaccoh** [comment]
One update: TV itself works. It's just the initial boot that doesn't work. So could we look into fixing Option 43 properly?

**Reply by deanhouben** [reply]
How did you fix the Error 4? I can't get it to work . Used the same config a you posted above.

**Reply by jaccoh** [reply]
> How did you fix the Error 4? I can't get it to work . Used the same config a you posted above.

I hooked up the supplied router from Solcon. Did all the updates and then switched back to my udm.

**Reply by deanhouben** [reply]
I got a problem that only regional channels are working now. So NL1/2/3 perfect. All the other channels not working. I hear sound but nothing on screen visible.
Did you do anything different in your config to get the other channels to work?

**Comment by jaccoh** [comment]
Nope. Just works.On 22 Sep 2023, at 13:20, deanhouben ***@***.***> wrote:﻿
I got a problem that only regional channels are working now. So NL1/2/3 perfect. All the other channels not working. I hear sound but nothing on screen visible.
Did you do anything different in your config to get the other channels to work?
